tools/spelling/spelling_dictionary.txt

```diff
@@ -167,6 +167,7 @@ LDS
 LEV
 LF
 LHS
+LLC
```

Comment:
revert most of the changes to this file, this has been fixed.

Reply:
ok